### ISSUE TITLE
Unused config parameters

### ISSUE THREAD

**Issue by metbril** [Perlkonig/grav-plugin-webmention#11]
Both the yaml and readme refer to

    url_name_map

https://github.com/Perlkonig/grav-plugin-webmention/blob/6607aaa1c0ed5713673a57766199dd2dd79b39e3/webmention.yaml#L3

and 

    file_url_name_map

(README line 84)

but I can't find anything elsewhere in the plugin code that uses those.

Am I missing something or are these deprecated and/or replaced within the sender/receiver parts?

**Comment by Perlkonig** [comment]
Good catch. I added those because they would have been useful for the planned module that actually displays your webmentions in your templates, but I decided ultimately not to bother with that end of things and leave it up to users to decide how they wanted to use that data. (There are just so many different ways one could approach it.) So the plugin doesn't use them. 

I should either remove them entirely or make this more clear in the README. Leaving this open to remind me to address this when I can.
